src/op_mode/qos.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2016-2024 VyOS maintainers and contributors
```

Comment:
2024 because this is a new script

Reply:
Do you mean just have "2024" instead of "2016-2024"?

Reply:
Yes, I do.

Reply:
Done!